src/migrations/20191115190415-course-settings.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async transaction => {
+      await queryInterface.addColumn(
+        'courses',
+        'isUnlisted',
+        {
+          type: Sequelize.BOOLEAN,
```

Comment:
CI is failing because the configuration here (specifically, default value and "allow null") is different from that in `src/models/Course`. I'll leave it up to you to figure out what the "right" config is, but the two should be in sync with each other.